vr-wg-charter.html

```diff
@@ -212,11 +212,11 @@ <h1 id="title">[DRAFT] WebVR Working Group Charter</h1>
           <h3>Timeline</h3>
 
             <ul>
-              <li>May 2017: First teleconference</li>
+              <li>Jul 2017: First teleconference</li>
```

Comment:
July-August might be challenging for many due to holiday season, so pushing the first call to late August-early September might work better.